src/nvim/ex_cmds.c

```diff
@@ -306,9 +306,12 @@ static int linelen(int *has_tab)
     ;
   save = *last;
   *last = NUL;
-  len = linetabsize(line);              /* get line length */
-  if (has_tab != NULL)                  /* check for embedded TAB */
-    *has_tab = (vim_strrchr(first, TAB) != NULL);
+  // Get line length.
+  len = linetabsize(line);
+  // Check for embedded TAB.
+  if (has_tab != NULL) {
+    *has_tab = (vim_strchr(first, TAB) != NULL);
```

Comment:
I think that you forgot to replace `vim_strrchr` with `STRRCHR` here :)

Reply:
Why change this, though?

Reply:
Because vim_strrchr needs to be replaced. Or maybe I misunderstood your question?

Reply:
Why not use strrchr? It's not worth having to analyze the code and find later that Vim did it for some reason. Even though it seems obvious that there is no reason, a near-negligible risk is not worth a near-negligible gain.